src/plot_utils.py

```diff
@@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
 
 
 def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
-        col_ordering=None):
+            col_ordering=None):
```

Comment:
Lint complains about that. Do you know how to set pylint up to know about
this convention?


Gregory A. Marton  617-858-0775
http://goo.gl/Ne09o
http://csail.mit.edu/~gremio


On Tue, Sep 29, 2015 at 2:56 PM, F Saad <notifications@github.com> wrote:

> In src/plot_utils.py
> <https://github.com/probcomp/bdbcontrib/pull/45#discussion_r40713233>:
>
> > @@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
> >
> >
> >  def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
> > -        col_ordering=None):
> > +            col_ordering=None):
>
> The preferred convention in bdbcontrib and bayeslite to keep indenting
> until no longer necessary (i.e. here we just need two indents so that
> col_ordering does not overlap with the docstring and the function code),
> rather than aligning the next line with the opening parenthesis.
>
> I made several such whitespaces changes when refactored bdbcontrib to
> regularize the codebase.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/probcomp/bdbcontrib/pull/45/files#r40713233>.
>